From 588af428d3a9b572caea909f9f5f4b00f4922ed3 Mon Sep 17 00:00:00 2001
From: Dillibabu <dillibabug@ami.com>
Date: Thu, 16 May 2024 13:03:22 +0530
Subject: [PATCH 1/1] Signed-off-by: Dillibabu <dillibabug@ami.com>

1. Changed power control gpio pins for x86-power-control, GPIO CHIP 0, PORTB, PINS 8,9,10
---
 arch/arm64/boot/dts/aspeed/ast2700-evb.dts | 28 ++++++++++++++++++++++
 1 file changed, 28 insertions(+)

diff --git a/arch/arm64/boot/dts/aspeed/ast2700-evb.dts b/arch/arm64/boot/dts/aspeed/ast2700-evb.dts
index 303e657c4356..e1dd9baaa325 100644
--- a/arch/arm64/boot/dts/aspeed/ast2700-evb.dts
+++ b/arch/arm64/boot/dts/aspeed/ast2700-evb.dts
@@ -386,6 +386,34 @@ &gpio0 {
 	pinctrl-0 = <&pinctrl_i3c12_driving &pinctrl_i3c13_driving
 		     &pinctrl_i3c14_driving &pinctrl_i3c15_driving>;
 	pinctrl-names = "default";
+
+	gpio-line-names =
+	/*A0-A7*/       "","","","","","","","",
+	/*B0-B7*/       "POWER_OUT","RESET_OUT","PS_PWROK","","","","","",
+	/*C0-C7*/       "","","","","","","","",
+	/*D0-D7*/       "","","","","","","","",
+	/*E0-E7*/       "","","","","","","","",
+	/*F0-F7*/       "","","","","","SIO_POWER_GOOD","","",
+	/*G0-G7*/       "","","","","","","","",
+	/*H0-H7*/       "","","","","","","","",
+	/*I0-I7*/       "","","","","","","","",
+	/*J0-J7*/       "","","","","","","","",
+	/*K0-K7*/       "","","","","","","","",
+	/*L0-L7*/       "","","","","","","","",
+	/*M0-M7*/       "","","","","","","","",
+	/*N0-N7*/       "","","","","","","","",
+	/*O0-O7*/       "","","","","","","","",
+	/*P0-P7*/       "","","","","","","","",
+	/*Q0-Q7*/       "","","","","","","","",
+	/*R0-R7*/       "","","","","","","","",
+	/*S0-S7*/       "","","","","","","","",
+	/*T0-T7*/       "","","","","","","","",
+	/*U0-U7*/       "","","","","","","","",
+	/*V0-V7*/       "","","","","","","","",
+	/*W0-W7*/       "ID_BUTTON","SIO_S5","SIO_ONCONTROL","POWER_BUTTON","NMI_OUT","RESET_BUTTON","NMI_BUTTON","POST_COMPLETE",
+	/*X0-X7*/       "","","","","","","","",
+	/*Y0-Y7*/       "","","","","","","","",
+	/*Z0-Z7*/       "","","","","","","","";
 };
 
 &i3c0 {
-- 
2.34.1

